examples/with-gatsby/packages/eslint-config-custom/package.json

```diff
@@ -1,25 +1,10 @@
 {
   "name": "eslint-config-custom",
+  "license": "MIT",
   "version": "0.0.0",
   "private": true,
-  "main": "index.js",
-  "license": "MIT",
-  "files": [
-    "next.js",
-    "gatsby.js"
-  ],
-  "dependencies": {
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.23.0",
-    "eslint-config-next": "13.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-config-turbo": "latest",
-    "eslint-plugin-gatsby": "^1.0.2",
-    "eslint-plugin-react": "7.31.8"
-  },
   "devDependencies": {
-    "@types/eslint": "^8.21.2",
-    "eslint-config-react-app": "^7.0.1",
-    "typescript": "^4.7.4"
+    "@vercel/style-guide": "^5.0.0",
```

Comment:
is this supposed to be here? seems like a weird thing to ship to customers in an example

Reply:
@mehulkar it is. We're using this instead of writing all our own rules / including another preset (like airbnb etc)